src/shared/DotnetTool/pack-tool.sh

```diff
@@ -53,36 +52,25 @@ GITLAB_UI_SRC="$SRC/shared/GitLab.UI.Avalonia"
 DOTNET_TOOL="shared/DotnetTool"
 PROJ_OUT="$OUT/$DOTNET_TOOL"
 
-PACKAGE="$ROOT/nuget"
-CONFIGURATION="${CONFIGURATION:=Release}"
+CONFIGURATION="${CONFIGURATION:=Debug}"
```

Comment:
Switched the default back to `Debug` from `Release`.

Thinking about it again, we should make these scripts easy to call locally in dev (where we'd want debug bits) since CI doesn't care how long command line args get 😁